src/fairseq2/models/nllb/builder.py

```diff
@@ -67,6 +69,9 @@ class NllbConfig:
     dropout_p: float
     """The dropout probability in Transformer layers."""
 
+    decoder_activate_fn: Module = ReLU()
```

Comment:
nit: rename this to `decoder_activation_fn`

Reply:
done, thanks